addons/mail/static/src/js/mail.js

```diff
@@ -850,6 +855,15 @@ var MailThread = Attachment.extend ({
         }
     },
 
+     /**
+     * Used for @ mention functionality
+     * Convert span tag into link, To make link of partner
+     */
+    convert_into_link: function(msg) {
```

Comment:
give param to this function, and extract it in `mail_mention.js`. This function should generic, receive a msg as input, and return the substituted message.